test/test_bulk.py

```diff
@@ -295,8 +295,8 @@ def test_upsert(self):
         self.assertEqual(self.coll.count_documents({'foo': 'bar'}), 1)
 
     def test_numerous_inserts(self):
-        # Ensure we don't exceed server's 1000-document batch size limit.
-        n_docs = 2100
```

Comment:
I don't remember why this was hard coded. Do we not get it from the server on some old version like 2.6?

Reply:
We always use max_write_batch_size internally, this test was never updated when the server updated the batch size from 1000 to 100000.